synapse/rest/media/v1/preview_url_resource.py

```diff
@@ -481,8 +483,10 @@ async def _expire_url_cache_data(self):
 
         await self.store.delete_url_cache_media(removed_media)
 
-        logger.info("Deleted %d media from url cache", len(removed_media))
-
+        if removed_media:
+            logger.info("Deleted %d media from url cache", len(removed_media))
+        else:
+            logger.debug("No media removed from url cache")
```

Comment:
Looks like this is failing style checking:

```suggestion
            logger.debug("No media removed from url cache")

```